From: Dominik George <nik@naturalnet.de>
Subject: Load Debian JS libraries
 We removed those libraries from the source that Debian has.
 We need to load them here; upstream originally would have
 compiled them into the minified main application file.
--- a/guacamole/src/main/webapp/index.html
+++ b/guacamole/src/main/webapp/index.html
@@ -54,7 +54,13 @@
                     help-text="loginHelpText"
                     form="expectedCredentials"
                     values="acceptedCredentials"></guac-login>
-        
+     
+        <script type="text/javascript" src="lib/jquery/jquery.min.js"></script>
+        <script type="text/javascript" src="lib/lodash.min.js"></script>
+        <script type="text/javascript" src="lib//angular/angular.min.js"></script>
+        <script type="text/javascript" src="lib//angular/angular-cookies.min.js"></script>
+        <script type="text/javascript" src="lib//angular/angular-route.min.js"></script>
+        <script type="text/javascript" src="lib//angular/angular-touch.min.js"></script>
         <script type="text/javascript" src="app.js?v=${project.version}"></script>
     </body>
 </html>
